### DISCUSSION TITLE
Throttle detents settings via flybag: also in developement versions?

### DISCUSSION THREAD

**Discussion by PatriceCesson** [flybywiresim/aircraft#4177] [General]
Hi,
Will the method for throttle detents settings used in the latest experimental version (via the flybag ) be deployed also in the developement versions?
Thanks.
Patrice.

**Comment by MapleNAV-TCASFAULT** [comment]
That will come when the custom AP branch is merged into the dev version or possibly when the developer of the throttle calibration page decides to make a PR for the master branch.

**Comment by PatriceCesson** [comment]
Ok. Thank you.
